packages/cli/src/databases/entities/User.ts

```diff
@@ -62,7 +62,7 @@ export class User {
 	@PrimaryGeneratedColumn('uuid')
 	id: string;
 
-	@Column({ length: 254 })
+	@Column({ length: 254, nullable: true })
 	@Index({ unique: true })
 	@IsEmail()
 	email: string;
```

Comment:
Attempting to type L68 as `string | null` triggers [this issue](https://github.com/typeorm/typeorm/issues/1358).